build.cake

```diff
@@ -237,7 +233,7 @@ Task("TestNetStandard20EngineCore")
     .OnError(exception => { UnreportedErrors.Add(exception.Message); })
     .Does(() =>
     {
-        RunDotnetNUnitLiteTests(NETCORE_ENGINE_CORE_TESTS, "netcoreapp2.1");
+        RunDotnetNUnitLiteTests(NETCORE_ENGINE_CORE_TESTS, "netcoreapp3.1");
```

Comment:
Update Task name and comments above.

Reply:
Actually, that's a really good catch! The Task name is what it's supposed to do, but it's no longer doing it. When running tests under .NET Core 2.1, the SUT was picked up as .NET Standard, but when running under 3.1 it picks up the 3.1 build of the engine core. So the two tasks TestNetStandard20EngineCore and TestNetCore31EngineCore actually do the same thing now!!! I'll have to fix that by making sure the tests pull in the correct SUT.

Reply:
Postponing this.